packages/contracts-bedrock/src/L1/L1ERC721Bridge.sol

```diff
@@ -28,8 +28,8 @@ contract L1ERC721Bridge is ERC721Bridge, ISemver {
     ISuperchainConfig public superchainConfig;
 
     /// @notice Semantic version.
-    /// @custom:semver 2.3.1
-    string public constant version = "2.3.1";
+    /// @custom:semver 2.4.0
+    string public constant version = "2.4.0";
 
```

Comment:
Why did the L1/L2ERC721Bridge's change? I don't see a change to the base ERC721Bridge